jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -304,11 +304,13 @@ YUI.add('deployment-flow', function() {
       return (
         <span className="deployment-flow__service-title">
           Applications to be deployed
-          <juju.components.GenericButton
-            action={this._toggleChangelogs}
-            type="base"
-            title={this.state.showChangelogs ?
-              'Hide changelog' : 'Show changelog'} />
+          <div className="right">
```

Comment:
The type adds logic. I thought the manipulation might sting someone later so better to keep the "type" stuff consistent. Happy to change if that is not a prob?